cmd/create/cluster/cmd.go

```diff
@@ -1262,7 +1262,7 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	if privateLink {
+	if privateLink || cmd.Flags().Changed("hosted-cp") {
```

Comment:
Not so sure as I'm no so familiar with this flow, but I feel like you can probably use this
https://github.com/openshift/rosa/blob/2a4b109a7080b54e2a4ce9cc68797487b57c99d7/cmd/create/cluster/cmd.go#L1207 instead of doing flags changed

Reply:
by the way privateLink should already include the isPrivateHostedCP I think
https://github.com/openshift/rosa/blob/2a4b109a7080b54e2a4ce9cc68797487b57c99d7/cmd/create/cluster/cmd.go#L1208

Reply:
@gdbranco isHostedCP only returns true when you answer `Yes` to the "? Deploy cluster with hosted control plane: [? for help] (Y/n)" question

When discussing this story with the assigner (Priya) she said that it should be as long as the `--hosted-cp` flag is used

Reply:
https://github.com/openshift/rosa/blob/d09fd9f9cb0ab4e865a60942bb418ac99b91d260/cmd/create/cluster/cmd.go#L600

Based on that I believe so, at least it is the answer to the prompt and it is also where the arg value is held as a variable